login1/dbus.go

```diff
@@ -45,6 +45,11 @@ func New() (*Conn, error) {
 	return c, nil
 }
 
+// Close closes the dbus connection
+func (c *Conn) Close() {
+	c.conn.Close()
```

Comment:
What about checking for `nil` on both `c` and `c.conn` so we are sure a deferred `c.Close()` never panics?

Reply:
fixed